cpu/esp32/Makefile.dep

```diff
@@ -38,6 +38,7 @@ ifneq (,$(filter esp_wifi_any,$(USEMODULE)))
     USEMODULE += esp_idf_wpa_supplicant_port
     USEMODULE += esp_idf_nvs_flash
     USEMODULE += riot_freertos
+    USEMODULE += xtimer
```

Comment:
This should also be added after this line to fix the Murdock failure:
https://github.com/RIOT-OS/RIOT/blob/4337cb316ff45b7891eedb68102ca07bde092719/cpu/esp32/Makefile.dep#L14

Reply:
Thanks for pointing that out. I didn't see that failure because I was compiling all the tests for the `esp32-wroom-32` board and not the `esp32-olimex-evb` board which enables `esp_eth` as network default device.

Reply:
Fixed